### ISSUE TITLE
Config option for disable SSL check

### ISSUE THREAD

**Issue by svenez** [2BAD/bitrix#21]
Sending request to host that has self-signed cert raises an error 

`GotError [RequestError]: self signed certificate`

any options to avoid this?

**Comment by mrhyde** [comment]
An option to override HTTP client behavior vas implemented in 8555802b7668896ff775a7bdd6b6f674464345e2 that will be published on NPM after I will run some more tests

**Comment by mrhyde** [comment]
Version 2.4.0 was published on NPM. Let me know if it works for you:
```typescript
const bitrix = Bitrix('https://PORTAL_NAME.bitrix24.ru/rest', 'ACCESS_TOKEN', {https: { rejectUnauthorized: false }})
```
